src/app/shared/NoteOptions.svelte

```diff
@@ -29,6 +42,14 @@
       <Field icon="fa-warning" label="Content warnings">
         <Input bind:value={values.warning} placeholder="Why might people want to skip this post?" />
       </Field>
+      {#if publishAt}
+        <Field icon="fa-hourglass-half" label="Schedule post">
+          <Input
+            type="datetime-local"
+            min={new Date().toISOString().slice(0, 16)}
+            bind:value={values.publish_at} />
+        </Field>
```

Comment:
We have a DateTime input that should simplify some of the parsing/formatting logic